nifi-commons/nifi-site-to-site-client/src/main/java/org/apache/nifi/remote/client/AbstractPeerPersistence.java

```diff
@@ -44,11 +44,19 @@ protected PeerStatusCache restorePeerStatuses(final BufferedReader reader,
             return null;
         }
 
+        final String remoteInstanceUris;
+        remoteInstanceUris = String.valueOf(reader.readLine());
```

Comment:
No need to call String.valueOf here, or the declare `remoteInstanceUris` separately. Can just be `final String remoteInstanceUris = reader.readLine();`

Reply:
I changed a little here, it will check if the pattern starts with our prefix. if yes, then assign the URIs to the variable.